### PR TITLE
[#2] [Chore] Set up iOS app in KMM SwiftUI project

### LINKED ISSUE #2: [Chore] Set up iOS app in KMM SwiftUI project
# Why
To testify and have a proper iOS project setup, we will have to use the iOS chapter's template to generate an iOS project inside the `iosApp` folder of the KMM project.

# Acceptance Criteria
- Use the iOS chapter's [template](https://github.com/nimblehq/ios-templates) to generate an iOS project.
- The new iOS project must have bundle ids
   - Production: `co.nimblehq.phong.kmmswiftuiic`.
   - Staging: `co.nimblehq.phong.kmmswiftuiic.staging`.

### PR DESCRIPTION
**Pull request by phongvhd93**
- Close #2 

## What happened 👀

- Use the iOS chapter template to generate an iOS application in `iosApp` folder.
- Set up provisioning profile and match configuration.
- Set stating bundle id and production bundle id

## Insight 📝

To keep the iOS project consistent with the project structure the iOS chapter is using, I replaced the default application generated by AndroidStudio with the project, which is generated by the iOS chapter's ios-template

## Proof Of Work 📹

https://user-images.githubusercontent.com/22606906/230257504-34c93c79-0bb8-4df8-8512-ed04845a6e75.mp4

### CODE REVIEW

**Review by markgravity — APPROVED** [review]

File: iosApp/.sourcery.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - PhongKMMIC/Sources
+templates:
+  - tools/sourcery/templates
+output:
+  PhongKMMICTests/Sources/Mocks/Sourcery 
+args:
+  autoMockableTestableImports:
+    - PhongKMMIC
+  autoMockableImports:
+    - RxSwift
```
Comment (suho) [inline iosApp/.sourcery.yml]: Please remove RxSwift and RxCocoa

File: iosApp/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (suho) [inline iosApp/LICENSE]: Please remove this file

File: iosApp/PhongKMMIC/Configurations/Plists/Info.plist [hunk 3]
```diff
@@ -13,22 +13,19 @@
 	<key>CFBundleName</key>
 	<string>$(PRODUCT_NAME)</string>
 	<key>CFBundlePackageType</key>
-	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
+	<string>APPL</string>
```
Comment (suho) [inline iosApp/PhongKMMIC/Configurations/Plists/Info.plist]: is it expected to change to `APPL` 🤔

File: iosApp/PhongKMMIC/Sources/Application/AppDelegate.swift [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+@main
+final class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
+    ) -> Bool {
+        let homeViewController = HomeViewController()
```
Comment (suho) [inline iosApp/PhongKMMIC/Sources/Application/AppDelegate.swift]: If you are setting the project with SwiftUI, please remove these setup

File: iosApp/deliverable_setup.sh [hunk 5]
```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```
Comment (suho) [inline iosApp/deliverable_setup.sh]: Please remove this file

**Review by suho — CHANGES_REQUESTED** [review]

**Review by markgravity — APPROVED** [review]

**Review by suho — APPROVED** [review]

**Pull request merged**